sdk/keyvault/Azure.Security.KeyVault.Administration/Directory.Build.props

```diff
@@ -0,0 +1,11 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--
+    Add any shared properties you want for the projects under this package directory that need to be set before the auto imported Directory.Build.props
+  -->
+
+  <PropertyGroup>
```

Comment:
Why not put this into the sdk/keyvault/Directory.Build.props to apply to all 4 shipping track 2 packages?